Vanilla-JS-Projects/Intermediate/Countdown-Timer/README.md

```diff
@@ -0,0 +1,70 @@
+<h1 align='center'><b>💥 Countdown-Timer 💥</b></h1>
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h3 align='center'>Tech Stack Used 🎮</h3>
+
+<div align='center'>
+
+  ![HTML5](https://img.shields.io/badge/html5-%23E34F26.svg?style=for-the-badge&logo=html5&logoColor=white)
+  ![CSS3](https://img.shields.io/badge/css3-%231572B6.svg?style=for-the-badge&logo=css3&logoColor=white)
+  <img width="55" src="https://raw.githubusercontent.com/gilbarbara/logos/master/logos/javascript.svg"/>
+
+</div>
+
+
+![Line](https://github.com/Avdhesh-Varshney/WebMasterLog/assets/114330097/4b78510f-a941-45f8-a9d5-80ed0705e847)
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h3 align='center'>Description 📃</h3>
+
+<div align='center'>
+    <p>This project is a simple vanilla javascript implementation of Countdown Timer with stopwatch and clock . This project have a COUNTDOWN TIMER  that counts down time from a given number with stop and reset option . Also this project have a CLOCK which displays time on the front of the webpage and a STOPWATCH with lap and reset option.
+      Its aim is to manage time.
+</p>
+</div>
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h3 align='center'>How to run this Project? 🕹️</h3>
+
+<div >
+    <p>To run this project locally, follow these steps:
+
+1. Fork the repository.
+
+2. Clone the repository to your local computer:
+    git clone https://github.com/your-username/Countdown-Timer-Clone.git
+
+3. Open the project folder in your preferred code editor, now you can view website in live.
+
+</p>
+</div>
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h3 align='center'>Screenshot📸</h3>
+1. <img src='./photo1.webp.webp'>
```

Comment:
Attach these images by uploading in the comment section and add their links here or you can refer from other project's readme files.